src/inventory_client/inventory_client.go

```diff
@@ -200,15 +200,15 @@ func (c *inventoryClient) DownloadHostIgnition(ctx context.Context, hostID strin
 	}()
 
 	params := installer.DownloadHostIgnitionParams{
-		ClusterID: c.clusterId,
+		ClusterID: strfmt.UUID(infraEnvID),
```

Comment:
Don't we have a new API that accepts InfraEnv ID as a parameter?

Reply:
not for downloading host ignitions

Reply:
I think that we should have, now that we are using InfraEnv to identify the hosts, all the APIs should be under /v2/infra-envs/<infraenvid>/hosts/<hostID>
we don't need new definitions in the swagger but if we want to introduce a new path in the swagger so the path params will be correct.

Reply:
adding another version: not changing anything in the download API, since it works at cluster level, changing UpdateHostInstallProgress to V2